Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -1025,16 +1025,16 @@ extension HTTPClient.Configuration {
         /// - warning: We highly recommend leaving this on.
         /// It is very common that connections establishment is flaky at scale.
         /// ``HTTPClient`` will automatically mitigate these kind of issues if this flag is turned on.
-        var retryConnectionEstablishment: Bool
+        public var retryConnectionEstablishment: Bool
 
         public init(idleTimeout: TimeAmount = .seconds(60)) {
             self.init(idleTimeout: idleTimeout, concurrentHTTP1ConnectionsPerHostSoftLimit: 8)
         }
 
-        public init(idleTimeout: TimeAmount, concurrentHTTP1ConnectionsPerHostSoftLimit: Int) {
+        public init(idleTimeout: TimeAmount, concurrentHTTP1ConnectionsPerHostSoftLimit: Int, retryConnectionEstablishment: Bool = true) {
```

Comment:
Unfortunately it's not API stable to add this extra parameter. Rather than us keep proliferating new `init`s, it might be nice for us to just add a zero-element initializer and encourage users towards setting the fields after default construction.

Reply:
Thanks for taking a look! I've...

- Reverted the change to the existing initializer.
- Added a parameterless initializer.
- Removed the default value for the parameter `idleTimeout` in the respective initializer (because it overlaps with the new parameterless initializer). Is that safe in terms of API stability?
- Moved the default values to the respective property declarations (to prevent them having to be specified at multiple points in the initializer code). This seems cleaner to me than initializer delegation, but if you prefer a different style, let me know.